stacker/stack.py

```diff
@@ -62,7 +62,8 @@ class Stack(object):
     def __init__(self, definition, context, variables=None, mappings=None,
                  locked=False, force=False, enabled=True, protected=False):
         self.name = definition.name
-        self.fqn = context.get_fqn(self.name)
+        self.fqn = definition.stack_name or context.get_fqn(self.name)
```

Comment:
Curious if anyone has thoughts on this. I intentionally made `stack_name` not include the include the namespace, but now I think that might be confusing. Would this be better?

```python
self.fqn = context.get_fqn(definition.stack_name or definition.name)
```

Reply:
Made this change.